tests/all.rs

```diff
@@ -34,6 +35,7 @@ pub mod rational_dimensions;
 
 #[test]
 #[cfg(feature = "f32")]
+#[cfg(feature = "f64")]
```

Comment:
Should the `f32` feature gate still be here? Nnot sure why it was there in the first place, since the test uses `f64`.

Reply:
You're right, removed.